ops/ccs-ops-misc/splunk/pipeline-job-failure-queries

```diff
@@ -0,0 +1,22 @@
+BFD-CCW-JOB-FAILED-PROD
+=======================
+index="577373831711" source="/bluebutton-data-pipeline/bluebutton-data-pipeline.log" tag::eventtype=error prod NOT prod-sbx NOT test jobType="gov.cms.bfd.pipeline.ccw.rif.CcwRifLoadJob"
+| where failure LIKE "%PipelineJobFailure%"
+| eval time=strftime(_time, "%H:%M:%S %m-%d-%y")
+| fields "_time", "time", "jobType", "failure"
+
+
+BFD-CCW-JOB-FAILED-PROD-SBX
+===========================
+index="577373831711" source="/bluebutton-data-pipeline/bluebutton-data-pipeline.log" tag::eventtype=error prod-sbx jobType="gov.cms.bfd.pipeline.ccw.rif.CcwRifLoadJob"
```

Comment:
why does this one not have the "NOT prod NOT test" similar to how the other two explicitly filter the other envs?

Reply:
So that part of the splunk query is effectively part of a 'dumb' text search. So if we had _NOT prod_, then _prod-sbx_ would get excluded. So we explicitly only match _prod-sbx_.

Now you may then ask, well why did you add _NOT prod_ and _NOT prod-sbx_ to the _test_ query. Probably could have gotten away with just test, but the more explicit one can be, the better off, since we don't know everyone's log message.

Reply:
Ok, makes sense!